README.md

```diff
@@ -15,6 +15,10 @@ You can also add your own custom metrics as an additional argument, or see the c
 
 General Logging service interface is defined in [frontend-platform](https://openedx.github.io/frontend-platform/module-Logging.LoggingService.html).
 
+## Deployment
+
+When you make changes, [draft a new release](https://github.com/edx/frontend-logging/releases) (and, if necessary, change the imported version number in importing packages).
```

Comment:
1. ohhh thanks, that's not happening in every repo we import as a package so I didn't realize
2. Yeah, but if I'm taking out (1) I might as well take out the bumping version reference, don't you think?

Reply:
Yeah, I agree with (2); I think the version bumping is also largely implied as a consumer of this package (for the most part?). I feel if we keep a "Deployment" section, it might largely serve to document that this repository uses Semantic Release to automate the release process (creating Git tags, creating GitHub releases, and publishing to NPM).